test/test_package.py

```diff
@@ -526,6 +527,68 @@ def test_inspect_class(self):
         regular_src = inspect.getsourcelines(regular_class)
         self.assertEqual(packaged_src, regular_src)
 
+    def test_package_fx_simple(self):
+        class SimpleTest(torch.nn.Module):
+            def forward(self, x):
+                return torch.relu(x + 3.0)
+
+        st = SimpleTest()
+        traced = symbolic_trace(st)
+
+        f = BytesIO()
+        with PackageExporter(f, verbose=False) as pe:
+            pe.save_pickle('model', 'model.pkl', traced)
+
+        f.seek(0)
+        pi = PackageImporter(f)
+        loaded_traced = pi.load_pickle('model', 'model.pkl')
+        input = torch.rand(2, 3)
+        self.assertTrue(torch.allclose(loaded_traced(input), traced(input)))
+
+    def test_package_then_fx(self):
+        from package_a.test_module import SimpleTest
+        model = SimpleTest()
+        f = BytesIO()
+        with PackageExporter(f, verbose=False) as pe:
+            pe.save_pickle('model', 'model.pkl', model)
+
+        f.seek(0)
+        pi = PackageImporter(f)
+        loaded = pi.load_pickle('model', 'model.pkl')
+        with set_module_env(ModuleEnv([pi, DefaultImporter])):
```

Comment:
This global thing is what I like the least about this approach. Basically, we need to poke in the `ModuleEnv` when doing symbolic tracing. The alternative of having to thread a ModuleEnv through all calls to `Graph()` is even less appealing though…